README.md

```diff
@@ -1,44 +1,41 @@
 <p align="center">
-<br />
-<a href="https://thirdweb.com"><img src="https://github.com/thirdweb-dev/js/blob/main/packages/sdk/logo.svg?raw=true" width="200" alt=""/></a>
-<br />
+    <br />
+    <a href="https://thirdweb.com">
+        <img src="https://github.com/thirdweb-dev/js/blob/main/packages/sdk/logo.svg?raw=true" width="200" alt=""/></a>
+    <br />
 </p>
-<h1 align="center">thirdweb JavaScript/TypeScript monorepo</h1>
+
+<h1 align="center"><a href='http://thirdweb.com/'>thirdweb</a> JavaScript/TypeScript monorepo</h1>
+
 <p align="center">
-<a href="https://github.com/thirdweb-dev/js/actions/workflows/CI.yml"><img alt="Build Status" src="https://github.com/thirdweb-dev/js/actions/workflows/CI.yml/badge.svg"/></a>
-<a href="https://discord.gg/thirdweb"><img alt="Join our Discord!" src="https://img.shields.io/discord/834227967404146718.svg?color=7289da&label=discord&logo=discord&style=flat"/></a>
+    <a href="https://github.com/thirdweb-dev/js/actions/workflows/CI.yml">
+        <img alt="Build Status" src="https://github.com/thirdweb-dev/js/actions/workflows/CI.yml/badge.svg"/>
+    </a>
+    <a href="https://discord.gg/thirdweb">
+        <img alt="Join our Discord!" src="https://img.shields.io/discord/834227967404146718.svg?color=7289da&label=discord&logo=discord&style=flat"/>
+    </a>
 </p>
+
 <p align="center"><strong>Best in class web3 SDKs for Browser, Node and Mobile apps</strong></p>
-<br />
 
-## Packages
+## Documentation
+
+Visit [https://portal.thirdweb.com/](https://portal.thirdweb.com/) to view the full documentation.
 
-| Package                                  | Description                                                          | Latest Version                                                                                                                                                                   |
```

Comment:
can we keep the packages list in the README? I think it's very useful

Reply:
ok

Reply:
done 👍